st2tests/st2tests/fixtures/generic/workflows/wf_has_jinja_st2kv_post_xform.yaml

```diff
@@ -0,0 +1,41 @@
+version: '2.0'
+
+demo:
+    type: direct
+    input:
+        - var1
+        - var2
+    tasks:
+        task_with_no_input:
+            action: st2.action
+            input:
+                ref: wolfpack.a1
+        task_with_inputs:
+            action: st2.action
+            input:
+                ref: wolfpack.a2
+                parameters:
+                    strtype: '{% raw %}{{ st2kv.system.var1 }}{% endraw %}'
+                    inttype: '{% raw %}{{ st2kv.user.var2 }}{% endraw %}'
```

Comment:
Just curious / confirming that user context (the user which executed the action) is indeed correctly passed down to Mistral.